.rubocop.yml

```diff
@@ -26,10 +24,6 @@ Layout/LineLength:
 Layout/MultilineMethodCallIndentation:
   Enabled: false
 
-Layout/MultilineHashBraceLayout:
-  Enabled: true
-  EnforcedStyle: new_line
```

Comment:
https://docs.rubocop.org/rubocop/cops_layout.html#layoutmultilinehashbracelayout shows that `symmetrical` rather than `new-line` is the default style, so can we keep this config around?

Reply:
Actually `symmetrical` seems fine, if that's what you intended. If the closing _and_ opening braces could require a new line I would prefer that, but `new_line` apparently allows this, and I'd definitely prefer `symmetrical` over this:

```
{ a: 1,
  b: 2
}
```

Reply:
Combination of Layout/FirstHashElementLineBreak (enabled) && Layout/MultilineHashBraceLayout (default: symmetrical) && others forces
```
{
  a: 1,
  b: 2,
}
```
… which was intended mostly to pull props into shape ;-) The config now shouldn’t change behavior though, at least there were just 2 auto-corrections.